contracts/vaults/RibbonThetaVault.sol

```diff
@@ -205,6 +257,21 @@ contract RibbonThetaVault is DSMath, GnosisAuction, OptionsVaultStorage {
         instantWithdrawalFee = newWithdrawalFee;
     }
 
+    /**
+     * @notice Sets the new discount on premiums for options we are selling
+     * @param newPremiumDiscount is the premium discount
+     */
+    function setPremiumDiscount(uint256 newPremiumDiscount)
+        external
+        onlyManager
+    {
+        require(newPremiumDiscount > 0, "newPremiumDiscount != 0");
```

Comment:
We should also set an upper bound for the discount to something reasonable like 30%

Reply:
👍